stackdriver/client.go

```diff
@@ -125,14 +125,11 @@ func (c *Client) getConnection(ctx context.Context) (*grpc.ClientConn, error) {
 		address = fmt.Sprintf("%s:%s", address, c.url.Port())
 	}
 	if c.resolver != nil {
-		conn, err := grpc.DialContext(ctx, c.resolver.Scheme()+":"+address, dopts...)
-		c.conn = conn
-		return conn, err
-	} else {
-		conn, err := grpc.DialContext(ctx, "dns:"+address, dopts...)
-		c.conn = conn
-		return conn, err
+		c.resolver.Scheme()
```

Comment:
This is a no-op. I assume this would read `address = c.resolver.Scheme() + ":" + address`. How does it pass unit tests?